Classes/Command/DoSomethingCommand.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace T3docs\Examples\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class DoSomethingCommand extends Command
+{
+    protected function configure(): void
+    {
+        $this->setDescription('A Command that does nothing and always fails');
```

Comment:
Setting the description is superfluous and redundant to the definition in Services.yaml. Maybe we can use setHelp() to preserve the configure() method.

Reply:
Done